### PR TITLE
feat: raise alert on long running queries per user instead of single pid

### PR DESCRIPTION
**Pull request by dcupif**
This commit modifies the existing alert on long running queries to be raised at the user level instead of the finer granularity of the pid level.

The reason is that if a query holds a lock which blocks several queries, we will raise an alert for each query on hold, therefore spamming our alert channels and creating noise.

Reducing the granularity to the user will improve readability and also understand quicker which apps and services are impacted.

### CODE REVIEW

**Comment by dcupif** [comment]
#sre-storage

**Review by jbourdale — APPROVED** [review]

**Comment by dcupif** [comment]
@vmercierfr this would be a breaking change since I modified the alert name. How would you like to handle the release of such change? I would create a new release, but we should adhere to semantic versioning standards I guess

File: charts/prometheus-postgresql-alerts/values.yaml [hunk 1]
```diff
@@ -77,14 +77,14 @@ rules:
...
+  PostgreSQLLongRunningQueries:
+    expr: max by (target, datname, usename) (pg_active_backend_duration_minutes{usename!=""}) > 30
     for: 1m
     labels:
       severity: warning
     annotations:
-      summary: "Long running query on {{ $labels.datname }} of {{ $labels.target }}"
-      description: "{{ $labels.usename }} is running a long query on {{ $labels.datname }} of {{ $labels.target }} with pid {{ $labels.pid }}"
+      summary: "Long running queries on {{ $labels.datname }} of {{ $labels.target }} initiated by {{ $labels.usename }}"
+      description: "{{ $labels.usename }} is running long queries on {{ $labels.datname }} of {{ $labels.target }}"
```
[the remark this answers is not in the document]
Comment (dcupif) [reply charts/prometheus-postgresql-alerts/values.yaml:87]: In this setup there are potential several `pid`, to be honest I'm unsure if I can access them, and how I could properly display it. But yes, if possible it'd be nice to have them all in the description I guess!

**Review by qfritz — APPROVED** [review]

**Comment by vmercierfr** [comment]
> @vmercierfr this would be a breaking change since I modified the alert name. How would you like to handle the release of such change? I would create a new release, but we should adhere to semantic versioning standards I guess

LGTM, we would release a v0.2.0-beta1 to be consistentt

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
> https://semver.org/

Meanwhile, I documented how to release versions: https://github.com/qonto/database-monitoring-framework/pull/30

**Review by vmercierfr — APPROVED** [review]

**Pull request merged**